### DISCUSSION TITLE
Cannot use captions in Lambda project

### DISCUSSION THREAD

**Discussion by TheFizz** [dlemstra/Magick.NET#815] [Help]
I need to generate an image within the AWS Lambda function. Everything is fine except this piece of code:

```
var captionSettings = new MagickReadSettings()
{
    FontFamily = "Times New Roman",
    Height = captionHeight,
    Width = captionWidth,

    FillColor = MagickColors.Magenta,
    BackgroundColor = MagickColors.Transparent,
};

using (var caption = new MagickImage($"caption:{image.Label}", captionSettings))
{
   // Doing stuff
}
```

I am getting `UnableToReadFont 'helvetica' @ error/annotate.c/RenderFreetype/1403`. The version in the test environment on a PC works just fine. This might be because Lambda has no fonts whatsoever. 

Can I do anything about it? Is there any workaround?

**Comment by dlemstra** [comment]
You will need to add the fonts to your application. If you only have a single font I would advise you to specifiy a path to your font instead: 

```C#
var captionSettings = new MagickReadSettings()
{
     Font = "/fonts/Times.ttf"
}
```

**Reply by TheFizz** [reply]
Hi, thanks for the info! However, Lambda is serverless, so It has no internal file structure. Can I somehow use the *.ttf file via stream?
